### PR TITLE
SearXNG

### LINKED ISSUE #9504: SearXNG
### Brand Name

SearXNG

### Website

https://docs.searxng.org/

### Popularity Metric

[GitHub stars: 5.2k](https://github.com/searxng/searxng/)

### Official Resources for Icon and Color

svg: https://docs.searxng.org/_static/searxng-wordmark.svg
color: `#3050ff`

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by jflattery**
**Issue:** closes #9504

Popularity Metric: [GitHub stars: 5.2k](https://github.com/searxng/searxng/)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I set the hex value to be the value of the icon as it appears on the site.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11629,6 +11629,11 @@
             "hex": "6EBE49",
             "source": "https://branding.seagate.com/productpage/3fc51aba-c35a-4eff-a833-a258b0440bd2"
         },
+        {
+            "title": "SearXNG",
+            "hex": "3050ff",
+            "source": "https://docs.searxng.org/_static/searxng-wordmark.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:11635]:
```suggestion
            "source": "https://docs.searxng.org"
```

We can get away with just linking to the homepage for this one.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11629,6 +11629,11 @@
             "hex": "6EBE49",
             "source": "https://branding.seagate.com/productpage/3fc51aba-c35a-4eff-a833-a258b0440bd2"
         },
+        {
+            "title": "SearXNG",
+            "hex": "3050ff",
```
Comment (adamrusted) [inline _data/simple-icons.json:11634]:
```suggestion
            "hex": "3050FF",
```

Our linter complains if the hex code isn't upper-case!

File: icons/searxng.svg
Comment (adamrusted) [inline icons/searxng.svg:1]: Can't accept this with JS baked into it. Please refer to our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and re-submit the icon to spec.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/searxng.svg
Comment (jflattery) [reply icons/searxng.svg:1]: Fixed!

**Comment by jflattery** [comment]
I believe I committed / accepted your changes and it should be good to go.

**Comment by adamrusted** [comment]
Looks like the SVG still has the JS attached to it, as well as a lot of additional attributes (would suggest it's not been optimized with SVGO yet). Could you please have another go @jflattery?

File: icons/searxng.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M105.926 53.91c0 16.562-13.432 29.994-30.009 29.994-16.562 0-29.994-13.432-29.994-29.994 0-16.577 13.432-30.01 29.994-30.01 16.577 0 30.01 13.433 30.01 30.01Zm0 0" style="fill:none;stroke-width:10;stroke-linecap:butt;stroke-linejoin:miter;stroke:#000;stroke-opacity:1;stroke-miterlimit:4" transform="translate(-10.675 -4.543) scale(.26086)"/><path d="M67.516 37.917a18.015 18.015 0 0 1 21.055 3.31c5.54 5.585 6.798 14.12 3.13 21.084" style="fill:none;stroke-width:5;stroke-linecap:butt;stroke-linejoin:miter;stroke:#000;stroke-opacity:1;stroke-miterlimit:4" transform="translate(-10.675 -4.543) scale(.26086)"/><path d="m12.992 16.79 3.403-3.552 7.527 7.211L20.523 24Zm0 0" style="stroke:none;fill-rule:nonzero;fill:#000;fill-opacity:1"/></svg>
```
Comment (adamrusted) [inline icons/searxng.svg:1]: Can see the issue you're having with the SVG - please find below optimized version put through [SVGOMG](https://jakearchibald.github.io/svgomg/) at 3 decimals. Feel free to use this if you want - but we'll need another @simple-icons/icon-review member to review 😄 

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>SearXNG</title><path d="m13.716 17.261 6.873 6.582L24 20.282l-6.824-6.536a9.11 9.11 0 0 0 1.143-4.43c0-5.055-4.105-9.159-9.16-9.159S0 4.261 0 9.316c0 5.055 4.104 9.159 9.159 9.159a9.11 9.11 0 0 0 4.557-1.214ZM9.159 2.773a6.546 6.546 0 0 1 6.543 6.543 6.545 6.545 0 0 1-6.543 6.542 6.545 6.545 0 0 1-6.542-6.542 6.545 6.545 0 0 1 6.542-6.543ZM7.26 5.713a4.065 4.065 0 0 1 4.744.747 4.064 4.064 0 0 1 .707 4.749l1.157.611a5.376 5.376 0 0 0-.935-6.282 5.377 5.377 0 0 0-6.274-.987l.601 1.162Z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by service-paradis — APPROVED** [review]
LGTM 🎉 Thank you @jflattery and @adamrusted!

**Pull request merged**